### PR TITLE
Refactor Code

### PR DESCRIPTION
**Pull request by wuyepabdul**
- Refactored code by replacing the objects and arrays with classes
- Added linter setup

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+/* eslint-disable max-classes-per-file */
+
+class Book {
```
Comment (Emmyn5600) [inline app.js:3]: - [x] Let us try to fix eslint without disabling line of code, You have used many classes in one, please find another way of correcting this issue.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @wuyepabdul and @M0rrighan 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
-  Git/GitHub flow best practices have been followed.:heavy_check_mark:
- Add, remove and localStorage are working properly:heavy_check_mark:
storag


Changes required :writing_hand: 

  - Kindly, follow github flow, you should have created a new branch called development and make it as a default branch to merge after getting approved
  - Please work on the Readme file to give the overview of the project you did.
   - Please check the comment under the review.

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: app.js (hunk 1 above)
Comment (wuyepabdul) [reply app.js:3]: Hi..please note that it is OK to have multiple classes for this milestone. Kindly find attached a screenshot of  the project requirements
code-review

**Review by AshakaE — APPROVED** [review]
### Hi @wuyepabdul  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**